### PR TITLE
Add an attribute config option and splitting on commas

### PR DESCRIPTION
**Pull request by iantrich**
This is my first pull-request ever for HA! Woot!

Hopefully it's up to snuff and I'm not missing any key aspects to how the formal process should be handled. Thanks!

### CODE REVIEW

File: beer-card/beer-card.js [hunk 1]
```diff
@@ -48,10 +49,10 @@ class BeerCard extends HTMLElement {
     const config = this._config;
     const root = this.shadowRoot;
     if (hass.states[config.entity]) {
-      const list = hass.states[config.entity].attributes["list"];
+      const list = hass.states[config.entity].attributes[config.attribute] + '';
```
Comment (ciotlosm) [inline beer-card/beer-card.js]: If you're trying to get it as a string, I think a nicer option could be
const list = `${hass.states[config.entity].attributes[config.attribute]}`;

**Comment by ciotlosm** [comment]
Looks pretty good. Make sure you increment correct VERSION number as well, and move the PR to be against the dev branch.

**Comment by iantrich** [comment]
Absolutely; so forgive my ignorance, do I then resubmit the pull request after making those changes?

**Comment by ciotlosm** [comment]
If it would have been against same branch, any new commits get automatically added.

**Comment by iantrich** [comment]
You know...if only I could read what is right here at the bottom...oh boy

**Comment by ciotlosm** [comment]
Maybe do a rebase against 'dev' first

**Comment by iantrich** [comment]
Comments addressed. Sorry for not taking the time to read the main page on contributions. Thanks for maintaining this repo, it is making this lovelace conversion absolutely amazing!

**Comment by ciotlosm** [comment]
@iantrich no worries. I'm happy it helps people and always welcome contributions.

**Pull request merged**